File: backend/src/main/java/cohappy/backend/model/dto/response/GetChoreDTO.java
package cohappy.backend.model.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GetChoreDTO {
    private String choreCode;
    private String assignedTo;
    private String assignedToName;
    private byte[] assignedToImage;
    private boolean completed;
    private String createdBy;
    private String houseCode;
    private String name;
    private String description;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/response/GetChoreDTO.kt
package cohappy.frontend.model.dto.response;

data class GetChoreDTO(
    val choreCode: String? = null,
    var assignedTo: String? = null,
    var assignedToName: String? = null,
    var assignedToImage: ByteArray? = null,
    var completed: Boolean = false,
    var createdBy: String? = null,
    var houseCode: String? = null,
    var name: String? = null,
    var description: String? = null
)
